apiary.apib

```diff
@@ -682,8 +682,8 @@ A list of objects containing the following parameters:
 
 |  Name | Description | Example |
 | ---------------| ------------| --------|
- agent_available | Set to true if there is a free agent available | false |
-| queued_sessions | the number of sessions in the queue. If session tag is specified, only sessions with that tag will be counted | 24 |
+ agent_available | Returns true if there is an online available agent | false |
+| queued_sessions | A number of the sessions in the queue. If session tag is specified, only sessions with that tag will be counted | 24 |
```

Comment:
me too :) my explanation is that it is not any session, but a group of the specific 'surfly' sessions. in that case the is needed (from what I remember). your ideas? :/